src/main/resources/templates/home.html

```diff
@@ -3,9 +3,37 @@
 <head>
     <title>Aggregator</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
+
 <body>
-<h1>AGGREGATOR</h1>
-test :)
+
+    <header>
+        <div id="Logo">
+            <a href="home.html" alt="Strona główna">Project MLEKO</a>
+        </div>
+
+                
+        <div class="dropdown">
+        <img class="dropbtn" src="img/burger_button2.jpg" alt="Menu">   
+          <div class="dropdown-content">
+            <a href="home.html">Strona główna</a>
+            <a href="#">Funkcjonalność 1</a>
+            <a href="login.html">Panel logowania</a>
+          </div>
+        </div>
+        <a href="https://github.com/projektMleko/aggregator">
```

Comment:
to jest spoko, bo odnosimy się do zewnętrznej strony, więc jawnie podajesz cały adres.